app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,15 +1,20 @@
 module Api
   module V1
-    class RentsController < ApplicationController
+    class RentsController < ApiController
+      rescue_from Pundit::NotAuthorizedError, with: :pundit_not_authorized_error
       include Wor::Paginate
 
       def index
         @rents = Rent.includes(:book, :user).where(user_id: params[:id])
+        @user  = User.find(params[:user_id])
+        authorize(@user)
```

Comment:
To get a list of available rents for an use you should use a policy scope instead of this: